scripts/ansible-playbook/k8s_deploy_fresh_machine/README.md

```diff
@@ -0,0 +1,171 @@
+# Install k8s with Ansible use root user on ubuntu 18.04
+
+
```

Comment:
@yanmo96 Do you have instruction in the README to show user how to run your ansible script?

Reply:
yes 👍